packages/block-library/src/paragraph/test/paragraph.perf-native.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, measurePerformance, screen } from 'test/helpers';
+
+/**
+ * WordPress dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import Paragraph from '../edit';
+
+describe( 'RichText Performance', () => {
+	it( 'performance is stable when typing text using the Paragraph block', async () => {
```

Comment:
Suggestion to balance clarity and brevity. Feel free to ignore. 
```suggestion
describe( 'Paragraph', () => {
	it( 'performance is stable when typing text', async () => {
```